src/AutoSplitImage.py

```diff
@@ -40,13 +41,25 @@ class AutoSplitImage:
     image_type: ImageType
     byte_array: MatLike | None = None
     mask: MatLike | None = None
+    texts: list[str]
     # This value is internal, check for mask instead
     _has_transparency = False
     # These values should be overriden by some Defaults if None. Use getters instead
     __delay_time: float | None = None
     __comparison_method: int | None = None
     __pause_time: float | None = None
     __similarity_threshold: float | None = None
+    __rect: list[int]
+    __ocr_comparison_methods: list[int]
```

Comment:
Same error and reason as https://github.com/Toufool/AutoSplit/pull/272/files/d1dfff05bd16f4056e462c8a3aaa90cd388f258e..f3c0e3eac3883d0616017d3b8b1182280edecac5#r1485591097